android/pytorch_android/src/main/java/org/pytorch/PyTorchAndroid.java

```diff
@@ -10,7 +10,7 @@
     if (!NativeLoader.isInitialized()) {
       NativeLoader.init(new SystemDelegate());
     }
-    NativeLoader.loadLibrary("pytorch_jni");
+    NativeLoader.loadLibrary("pytorch_jni_lite");
```

Comment:
@cccclai 
Excuse me for interrupting.

By changing library to read, the following methods now throw an exception.
- loadModuleFromAsset

`NativePeer`  class expects [pytorch_jni](https://github.com/pytorch/pytorch/blob/master/android/pytorch_android/src/main/java/org/pytorch/NativePeer.java#L10).